include/zenoh/api/reply.hxx

```diff
@@ -84,6 +84,16 @@ class Reply : public Owned<::z_owned_reply_t> {
         return {};
     }
 #endif
+
+    /// @brief Construct a a shallow copy of this Reply.
+    /// This can be used, to send reply replies outside of ``Queryable`` callback.
```

Comment:
the documentation is wrong, since it is describes the behavior of Query clone.

Reply:
Can you help me with right documentation here ?

Reply:
I think  just `/// @brief Construct a a shallow copy of this Reply.` should be good enough.

Reply:
Sorry, could you also remove an extra `a` before `shallow_copy` ?